tutor/resources/styles/components/top-nav-bar.less

```diff
@@ -1,12 +1,43 @@
-@tutor-navbar-height: 61px;
+@tutor-navbar-height: 60px;
 @tutor-navbar-padding-horizontal: 4rem;
 
 // the @navbar-XX vars are bootstrap defaults
 
 .tutor-top-navbar {
+
+  background: @tutor-white;
+  box-shadow: 0 5px 5px 0 rgba(0, 0, 0, 0.1);
+  color: @openstax-gray;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  height: @tutor-navbar-height;
+  padding-right: @navbar-padding-horizontal;
+  padding-left:  @navbar-padding-horizontal;
+  z-index: @zindex-navbar-fixed;
+
+  .tutor-nav-controls {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 15px;
+    width: 1200px;
+  }
```

Comment:
1170px content width is also in the spec. 1200px with 15px gutters is a standard format we're using on OSweb for general content. We might be looking at widening the rest of the content.